src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -944,7 +944,7 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
   }
 
   object ValDef extends ValDefDeconstructor {
-    def _1: Modifiers = field.mods
+    def _1: Modifiers = tpd.Modifiers(field.symbol)
```

Comment:
Note that modifiers will be re-created as a new object everytime mods is called. Is it possible to split accessor into two accessors, one for flags and another for annotations? If not, an alternative would be to let Symbol inherit from Modifiers.